#!/usr/bin/env python
PACKAGE = "plane_detection_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("dist_tresh",    double_t,    0,   "Distance threshold for plane detection",   0.01,     0.005,  0.5)
gen.add("lim",    double_t,    0,   "PointCloud Box limit",   2,  0.5,  10)
gen.add("param_opt",     bool_t,    0,     "Enable parameter optimization", True)


exit(gen.generate(PACKAGE, "plane_detection_ros", "PlaneDetectionParameters"))